runtime.txt

```diff
@@ -1 +1 @@
-python-3.8.12
+python-3.12
```

Comment:
I don't think this file is used at all. We can probably remove it. I think this is from when we were testing on Heroku years ago.